src/components/Navbar.module.css

```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;700&family=Roboto+Slab:wght@100;400;700&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 30px;
+  height: 50px;
+}
+
+.nav div h1 {
+  font-size: 1.9rem;
+  width: fit-content;
+  height: 2.313rem;
+  font-family: Montserrat, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  cursor: pointer;
+}
+
+.nav div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.ul {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+  width: 50%;
+  margin-right: 30vw;
+}
+
+.nav svg {
+  width: 1.063rem;
+  height: 1.063rem;
+  color: #0290ff;
+  cursor: pointer;
+}
+
+.ul li a {
+  font-family: Montserrat, 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  color: black;
+  font-size: 0.813rem;
+  font-weight: normal;
+  width: 3.688rem;
+  height: 1rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+
+  /* border: 2px solid green; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: